spec/comparators/hashed_configuration_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../spec_helper'
+
+describe HashedConfiguration do
```

Comment:
The distinction is done in order to make writing the test for the end-user simpler.